harness/determined/cli/master.py

```diff
@@ -3,18 +3,38 @@
 
 import determined.cli.render
 from determined import cli
-from determined.common import api, yaml
+from determined.cli.errors import CliError
+from determined.common import yaml
 from determined.common.api import authentication, bindings
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
 
 @authentication.required
 def config(args: Namespace) -> None:
-    response = api.get(args.master, "config")
+    if args.log:
+        if args.level and args.no_color is not None:
+            master_config = bindings.v1Config(
+                log=bindings.v1LogConfig(level=args.level, color=False)
+            )
+        elif args.level:
+            master_config = bindings.v1Config(
+                log=bindings.v1LogConfig(level=args.level, color=True)
+            )
```

Comment:
My apologies for not thinking of this while you and I discussed the API before, but I just realized that if somebody has color turned off and does
```
det master config --log --level debug
```
it will turn color on, which may not be what users expect. Perhaps we should allow commands like
```
det master config --log --level debug --color off
det master config --log --color on
det master config --log --level debug
```
This would unfortunately necessitate updating the FieldMask handling code in the endpoint handler as well, to handle the paths `log`, `log.color`, and `log.level`. What do you think, @salonig23?

Reply:
I agree, and I am sorry for the oversight I should have thought of that. I will make the changes!

Reply:
Done!